src/styles/components/_form.scss

```diff
@@ -353,7 +355,30 @@ form {
     }
   }
   
-  
+  .checkbox-size-1 {
```

Comment:
can you name these classes descriptively, like xs, s, m, L?

Reply:
done